packages/fetch-api/README.md

```diff
@@ -98,6 +98,24 @@ const { data: gifs } = await gf.gifs(['3oEjHGr1Fhz0kyv8Ig'])
 const { data: gifs } = await gf.gifs('tv', 'arrested-development')
 ```
 
+## _animate_
+
+Create animated text gifs dynamicaly based on the text input. This endpoint will require you to [create a new api key](https://developers.giphy.com/dashboard/)
```

Comment:
Tiniest request here
@giannif Could you make this `create a new SDK key` instead of `create a new API key`?
We're only adding the ACLs for SDK keys.

Reply:
@shwetaatgiphy done 👍